neurovault/apps/statmaps/tasks.py

```diff
@@ -0,0 +1,63 @@
+from __future__ import absolute_import
+
+import os
+import numpy
+import pandas as pd
+from celery import shared_task
+import nibabel as nib
+from nilearn.plotting import plot_glass_brain
+from nilearn.image import resample_img
+from neurovault.apps.statmaps.models import Image
+import neurovault
+
+@shared_task
+def generate_glassbrain_image(nifti_file,pk):
+  nifti_file = str(nifti_file)
+  png_img_name = "glass_brain_%s.png" % pk
+  png_img_path = os.path.join(os.path.split(nifti_file)[0],png_img_name)
+  glass_brain = plot_glass_brain(nifti_file)
+  glass_brain.savefig(png_img_path)
+
+
+@shared_task
+def make_correlation_df(resample_dim=[4,4,4],pkl_path=None):
+  if not pkl_path:
+    pkl_path = os.path.abspath(os.path.join(neurovault.settings.PRIVATE_MEDIA_ROOT,'matrices/pearson_corr.pkl'))
+  # Get standard space brain
+  reference = os.path.join(os.environ['FREESURFER_HOME'],'subjects', 'fsaverage', 'mri', 'brainmask.nii.gz')
```

Comment:
This file does not exists - shouldn't this be brain.nii.gz like in https://github.com/NeuroVault/NeuroVault/blob/683763f6c5e8a2d80e94ded7761405614feb87c7/neurovault/apps/statmaps/utils.py#L92